include/controller.h

```diff
@@ -195,6 +195,7 @@ extern "C" {
 #endif
 
 void controller_init( void );
+void controller_exec_pif( const void *inblock, void *outblock );
```

Comment:
Maybe this should be called joybus_exec() and moved to joybus.c file, together with a non-installed joybus.h header, included by controller.c and rtc.c

Reply:
I'm totally on-board with this 👍🏻

Reply:
Addressed in https://github.com/DragonMinded/libdragon/pull/152/commits/d198b72f90aa4b7d832146d3d78998e99a0db48b